backend/data/models/Collection.js

```diff
@@ -0,0 +1,22 @@
+const connection = require("../connection");
+const { Sequelize } = connection;
+const { UUID, UUIDV4 } = Sequelize;
+
+const Collection = connection.define("collection", {
+  id: {
```

Comment:
this is not a blocking comment. i think that we can use the default integer ID for tables where we don't really query/display by the ID. like this one where we'd mostly query using the creator or business id.